docs/services/frame-system/_index.md

```diff
@@ -151,9 +153,9 @@ Configure the reference frame as follows:
 | Parameter | Required? | Required |
 | --------- | ----------- | ----- |
 | `parent`  | **Required** | Default: `world`. The name of the reference frame you want to act as the parent of this frame. |
-| `translation` | **Required** | Default: `(0, 0, 0)`. The coordinates that the origin of this component's reference frame has within its parent reference frame. <br> Units: _mm_. |
+| `translation` | **Required** | Default: `(0, 0, 0)`. The coordinates that the origin of this component's reference frame has within its parent reference frame. <br> Units: m in Frame Editor, mm in JSON. |
```

Comment:
this is indeed the case that the units are different in the editor-- confirmed with michael parks. a bit odd but he said he would make a ticket to mark units on the editor so at least that will be more clear

Reply:
good catch